package.json

```diff
@@ -80,9 +79,10 @@
     "angular-ui-bootstrap": "~2.5.0",
     "angular-ui-router": "~0.3.2",
     "angular-waypoints": "~2.0.0",
+    "angular": "1.5.11",
```

Comment:
it seems that the `package1-name` and `package1-name-subname` ... changed the sorting order (shortest became last). Where do these changes come from? I see plenty of these around.

Reply:
No idea 🤷‍♂️ and it seems to have sorted itself with one more `npm install react --save` weird are the ways of `npm` sometimes!